### PR TITLE
chore(tools/release): validate HTML before release

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by sidvishnoi** [comment]
I think https://github.com/w3c/respec/pull/3473 should be enough.

**Comment by marcoscaceres** [comment]
Not sure, the validator also gets updated periodically - it could happen between a merge and a release. I'd rather have this check in to be as sure as we can be.

File: tools/release.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ const { exec } = require("child_process");
 const loading = require("loading-indicator");
 const MAIN_BRANCH = "develop";
 const DEBUG = false;
+const vnu = require("vnu-jar");
+const tmp = require("tmp");
```
Comment (sidvishnoi) [inline tools/release.js:11]: Can we avoid adding this dependency (and its dependency) for something Node.js allows easily?

Comment (sidvishnoi) [reply tools/release.js:11]:
```js
const tempFile = path.join(os.tmpdir(), "index.html");
```

**Review by sidvishnoi — APPROVED** [review]

**Pull request merged**